Explorer/ProjectSettings/QualitySettings.asset

```diff
@@ -137,7 +137,7 @@ QualitySettings:
     realtimeGICPUUsage: 25
     adaptiveVsyncExtraA: 0
     adaptiveVsyncExtraB: 0
-    lodBias: 0.75
```

Comment:
Are those values correct?

Reply:
That's what Unity set by default - I just opened the project and removed the splash image. This is also always set to this value for me whenever I make a local build, so I'm assuming it's ok 🤔